rewrite-java-11/src/main/java/org/openrewrite/java/isolated/ReloadableJava11ParserVisitor.java

```diff
@@ -1655,7 +1655,7 @@ public J visitWildcard(WildcardTree node, Space fmt) {
             return null;
         }
         try {
-            String prefix = source.substring(cursor, max(cursor, getActualStartPosition((JCTree) t)));
+            String prefix = source.substring(cursor, indexOfNextNonWhitespace(cursor, source));
```

Comment:
Seems like this should do the same, no?
```suggestion
            String prefix = whitespace();
```

Reply:
I thought the same, but no. The `whitespace()` function calls also the `format` function, which does not work in combination with custom `formatWithCommentTree` function. It breaks when there are comments used:

```java
private Space whitespace() {
  int nextNonWhitespace = indexOfNextNonWhitespace(cursor, source);
  if (nextNonWhitespace == cursor) { return EMPTY; }
  Space space = format(source, cursor, nextNonWhitespace);  //  <- this call right here  
  cursor = nextNonWhitespace;
  return space;
}
```

If I would change it to `String prefix = whitespace()..getWhitespace();` the _helper_ test in the LombokTest class will break for example:

<img alt="image" src="https://github.com/user-attachments/assets/d3fb374a-55ef-4926-922f-fbe6d0186790" />